cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/statements/expressions/CallExpression.kt

```diff
@@ -45,7 +45,8 @@ import org.neo4j.ogm.annotation.Relationship
  * An expression, which calls another function. It has a list of arguments (list of [Expression]s)
  * and is connected via the INVOKES edge to its [FunctionDeclaration].
  */
-open class CallExpression : Expression(), HasType.TypeObserver, ArgumentHolder {
+open class CallExpression :
+    Expression(), HasType.TypeObserver, HasArgumentsAndOptionalBase, ArgumentHolder {
```

Comment:
I don't know why the `CallExpression` should suddenly have a `base`. If I understood it correctly, this type of node is used for function calls, i.e., calls which * never* have a base. If a base is required, I thought we used the `MemberCallExpression`. Similarly, an operator doesn't make sense then